src/Angor/Client/Pages/View.razor

```diff
@@ -748,5 +742,16 @@ else
         string sanitizedInput = HtmlStripperService.StripHtmlTags(input);
         return new MarkupString(sanitizedInput);
     }
+    
+    public bool CanInvest(Project project)
+    {
+        if (project == null) return false;
+
+        var now = DateTime.UtcNow;
+        if (now < project.ProjectInfo.StartDate) return false; // Investing period hasn't started
```

Comment:
The original thought was because both in /view and /invest we should have the same conditions for someone who can invest that we should have a function (created it as a service) that both places use that determines it.
And that way if we want to add a condition, we will just add it in one place.

Also, this way it's more readable to understand what the conditions are to invest, and not scramble around the code trying to find all the conditions (right now there are little, but maybe in the future there will be more?)